api/src/main/java/me/lokka30/treasury/api/economy/account/AccountData.java

```diff
@@ -74,14 +75,14 @@ public boolean isPlayerAccount() {
     /**
      * If the current {@link AccountData} object represents a
      * {@link NonPlayerAccount non-player account}, this method
-     * will return the identifier of the account. Otherwise, an empty
+     * will return the {@link NamespacedKey} identifier of the account. Otherwise, an empty
      * {@link Optional} will be returned.
      *
      * @return identifier or empty optional if the data is a player account data
      * @since 2.0.0
      */
     @NotNull
-    public Optional<String> getIdentifier() {
+    public Optional<NamespacedKey> getIdentifier() {
```

Comment:
It may be better to have the methods named as so:
- `Optional<NamespacedKey> getNonPlayerIdentifier()`
- `Optional<UUID> getPlayerIdentifier()`